lib/ansible/modules/cloud/cloudstack/cs_rootpassword.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2018, Gregor Riepl <onitake@gmail.com>
+# based on cs_sshkeypair (c) 2015, René Moser <mail@renemoser.net>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: cs_rootpassword
+short_description: Allows resetting VM root passwords on Apache CloudStack based clouds.
+description:
+    - Reset the root password on a virtual machine.
+    - Requires cloud-init installed in the virtual machine.
+version_added: '2.8'
+author: "Gregor Riepl (@onitake)"
+options:
+  vm:
+    description:
+      - Name of the virtual machine to reset the root password on.
+    required: true
+  domain:
+    description:
+      - Name of the domain the virtual machine belongs to.
+    required: false
+    default: null
+  account:
+    description:
+      - Account the virtual machine belongs to.
+    required: false
+    default: null
+  project:
+    description:
+      - Name of the project the virtual machine belongs to.
+    required: false
+    default: null
+  poll_async:
+    description:
+      - Poll async jobs until job has finished.
+    default: yes
+    type: bool
+extends_documentation_fragment: cloudstack
+'''
+
+EXAMPLES = '''
+# reset and get the root password
+- local_action: cs_rootpassword name=myvirtualmachine
```

Comment:
I would prefer yaml style here and it should be `vm` not `name`. Also please add a name to the task, instead of a comment.